distro-source/build.xml

```diff
@@ -31,10 +31,8 @@
       <!-- jars -->
       <zipfileset file="${gwt.build.lib}/gwt-dev.jar" prefix="${project.distname}" />
       <zipfileset file="${gwt.build.lib}/gwt-user.jar" prefix="${project.distname}" />
-      <zipfileset file="${gwt.build.lib}/gwt-servlet.jar" prefix="${project.distname}" />
```

Comment:
Note to reviewers: added this wildcard to handle the existing 3 jars plus the two new -sources jars, in the same way that we wildcard requestfactory jars below.